core/src/css/component/battlegrounds/in-game/bgs-opponent-overview-big.component.scss

```diff
@@ -106,11 +106,11 @@
 		width: 170px;
 		display: flex;
 		flex-direction: column;
+		justify-content: space-around;
```

Comment:
Would you mind posting a screenshot of what effects these changes have on the main BG window (the one that appears in-game with Alt + B)? Mainly the "Opponents" tab.

Thanks :)

Reply:
![2022-08-29_00-43-19](https://user-images.githubusercontent.com/43519401/187095840-ad9a4b70-183e-4b8d-976f-608ae2e194fa.png)
![2022-08-29_00-45-48](https://user-images.githubusercontent.com/43519401/187095816-2280a253-f81f-4b14-8818-f7b8e8960152.png)

Reply:
Actually, space-evenly will be better here.